### ISSUE TITLE
DRAWING_FILE_SIZE_TOO_LARGE has no test

### ISSUE THREAD

**Issue by werk24** [W24-Service-GmbH/werk24-python#38]
**Describe the bug**
The DRAWING_FILE_SIZE_TOO_LARGE Exception is not tested

**Comment by werk24** [comment]
unittest is present; integration test is missing

```
    async def test_uploading_huge_file(self) -> None:
        """ Huge file produces DRAWING_FILE_SIZE_TOO_LARGE?
        """

        client = W24TechreadClient.make_from_env()
        asks: List[W24Ask] = [W24AskVariantCAD(is_training=True)]

        async with client as session:

            # mock the side effect
            session._techread_client_https.upload_associated_file \
                = mock.MagicMock(side_effect=RequestTooLargeException)

            # trigger
            message = await session.read_drawing(b"", asks=asks).__anext__()

            # assert
            self.assertEqual(
                message.exceptions[0].exception_type,
                W24TechreadExceptionType.DRAWING_FILE_SIZE_TOO_LARGE)
```

**Comment by werk24** [comment]
Closed with #39
